app/templates/services/previous_services.html

```diff
@@ -84,41 +82,53 @@
   {% else %}
     {% include "partials/service_warning.html" %}
 
-    <h1 class="govuk-heading-l">Previous {{ lot.name|lower }} services</h1>
-
-    {{ summary.heading("Your services from {}".format(source_framework.name)) }}
-    {% if previous_services_still_to_copy|length > 1 %}
-      {{ summary.top_link("Add all your services", url_for('.confirm_copy_all_previous_services', framework_slug=framework.slug, lot_slug=lot.slug)) }}
+<h1 class="govuk-heading-l">Previous {{ lot.name|lower }} services</h1>
+    {{ govukButton({
+    'text': 'Add all your services',
```

Comment:
I wonder if this is clear enough or whether we want it to read "Add all your {source_framework} services".

Might be a long button for "Add all your Digital Outcomes & Specialists 4 services", but I think that could only ever apply on User Research Studios, since the others all have a single limit.

Reply:
Not sure if that makes sense as you're only copying from one lot?

`Add all your {source_framework} {lot.name} services` ?

Reply:
Hmm, I think we should keep this as it is now. We can see if this impacts anything once we open these frameworks for real - my unproven thought is that it won't.

Reply:
OK, I've reverted this now in 3ebdd88.